Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:18-alpine AS development
+
+WORKDIR /app
+
+COPY package*.json /app/
+
+RUN npm ci
+
+COPY . .
+
+EXPOSE 5173
+
+CMD npm run docker:dev
```

Comment:
The node build will use the command: `node build`

Reply:
If we are using this for production, we can add a second stage and copy the output folder.

After the node build command add this:

```Dockerfile
FROM node:18-alpine

COPY --from=builder ./build .

ENV NODE_ENV=production

EXPOSE 3000

CMD node build

```

Reply:
Good catch. We still need the `npm run build` command, but we only need to copy over the `build` folder, not the whole app source code.